README.md

```diff
@@ -93,6 +94,18 @@ spnr
 vpr
 ```
 
+You can check if you have the toolchain environments correctly installed as
+follows:
+```
+$ python3 fpgaperf.py --check-env --toolchain vpr
```

Comment:
What are your thoughts on --check-env? I was toying around with this since the dependencies could get quite onerous for this project. Is this useful and/or the right thing to do?

Reply:
I was *super* useful for me. I really liked it.

Reply:
It seems great to me!